README_LINUX.md

```diff
@@ -1,211 +1,199 @@
+#### Install location
 
-   ```
-   $> make install
-   ```
+By default, SuperCollider installs in `/usr/local`, a system-wide install. Maybe you can't or don't want to use superuser privileges, or just want to install for a single user. To do so, set `CMAKE_INSTALL_PREFIX` to the desired installation directory. One good place to put it would be `$HOME/usr/local`:
 
-   For the above step, you will probably need super-user privileges,
-   e.g. using "sudo".
+    cmake -DCMAKE_INSTALL_PREFIX=~/usr/local ..
 
-   Also, please run
+Make sure the `bin` subdirectory (in this case, `~/usr/local/bin`) is in your `PATH` if you do this.
 
-   ```
-   $> sudo ldconfig
-   ```
-   after installing for the first time.
+#### Speeding up repeated builds
 
- - to uninstall:
+If you are developing SC or you're constantly pulling in the latest changes, rebuilding SC repeatedly can be a drag. Installing `ccache` can speed up re-compilation. Here is how to configure cmake to use it:
 
-   ```
-   $> make uninstall
-   ```
+    cmake -DCMAKE_CXX_COMPILER=/usr/lib/ccache/g++ -DCMAKE_C_COMPILER=/usr/lib/ccache/gcc ..
 
-### Qt GUI
+This assumes your ccache executables are installed into `/usr/lib/ccache` - you may need to change the path to reflect your installation.
 
-By default, the Qt GUI support will be built into sclang. 
-If you want to build without it configure cmake like this:
+#### Library suffix
 
-```
-$> cmake -DSC_QT=OFF ..
-```
+In some situations, it is preferable to install libraries and plugins
+not in the `lib` directory but in a suffixed one, e.g. `lib64`.
+In such a case you can set the cmake variable `LIB_SUFFIX`.
+For example if you wish to install into `lib64`:
 
-Note: running headless SC in an X-less environment requires
-jackd without D-bus support. On Raspbian Jessie this requires
-compiling jackd rather than using the packaged version.
-Also, note that you will get errors on sclang startup from
-classes requiring Qt. A workaround and more details are 
-described in:
+    cmake -DLIB_SUFFIX=64 ..
 
-    http://supercollider.github.io/development/building-raspberrypi 
+### Step 3: Build
 
+If CMake ran successfully without errors, you are ready to move on to building. You can freely alternate between building and setting CMake flgs.
 
+After setting your CMake flags, just run
 
-### Speeding up repeated builds
+    make
 
-If you build SuperCollider repeatedly, we recommend installing `ccache`
-which can speed up re-compilation. Here is how to configure cmake to use it:
+And to install, run
 
-```
-$> cmake -DCMAKE_CXX_COMPILER=/usr/lib/ccache/g++ -DCMAKE_C_COMPILER=/usr/lib/ccache/gcc ..
-```
+    make install
 
-This assumes your ccache executables are installed into `/usr/lib/ccache` - you may need to change the path to reflect your installation.
+You will need to use `sudo make install` if you are doing a system-wide installation, which is the default.
+
+After installing for the first time, please run
+
+    sudo ldconfig
```

Comment:
Maybe explain why this command and what it does?

Reply:
I don't really know myself...

Reply:
i don't know either. might be about libscsynth.

Reply:
:)

Reply:
@snappizz @patrickdupuis `ldconfig` rebuilds your system library cache. Sometimes needed after (re)building an app with new library versions..